### PR TITLE
GCP Packer build

### PR DESCRIPTION
**Pull request by bhale**
Requirements:
-  set GCP_PROJECT_ID
-  provide an account.json with a service account that is a compute editor
- your ssh_authorized_keys: added to user-data

The image name is a bit dumb because GCP does not accept spaces or dots.

### CODE REVIEW

**Comment by dweomer** [comment]
Hi @bhale,

Thank you for this contribution. Can you update it to include the newly-released [v0.3.0](https://github.com/rancher/k3os/releases/tag/v0.3.0)?

**Comment by bhale** [comment]
@dweomer Updated.

File: package/packer/gcp/template.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "k3os_version": "v0.3.0",
+    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.3.0/k3os-amd64.iso"
+  },
+  "builders": [
+    {
+      "type": "googlecompute",
+      "account_file": "account.json",
+      "project_id": "{{user `gcp_project_id`}}",
+      "source_image": "ubuntu-1804-bionic-v20190911",
+      "zone": "us-east1-d",
```
Comment (dweomer) [inline package/packer/gcp/template.json]: Can `zone` be parameterized with the `region` var?

Comment (bhale) [reply package/packer/gcp/template.json]: Of course. Done.

**Pull request merged**